EvidenceApi.Tests/V1/E2ETests/EvidenceRequestsTest.cs

```diff
@@ -151,5 +152,143 @@ public async Task ReturnNotFoundWhenCannotFindEvidenceRequest()
             responseFind.StatusCode.Should().Be(404);
         }
 
+        [Test]
+        public async Task CanGetEvidenceRequestsWithValidService()
+        {
+            var expected = BuildEvidenceRequestsListWithSameResident();
+            var serviceRequestedBy = expected[0].ServiceRequestedBy;
+            var uri = new Uri($"api/v1/evidence_requests?serviceRequestedBy={serviceRequestedBy}", UriKind.Relative);
+            var response = await Client.GetAsync(uri).ConfigureAwait(true);
+            var json = await response.Content.ReadAsStringAsync().ConfigureAwait(true);
+            var result = JsonConvert.DeserializeObject<List<EvidenceRequestResponse>>(json);
+
+            response.StatusCode.Should().Be(200);
+            result.Should().BeEquivalentTo(expected[0]);
```

Comment:
shouldn't `result` be a List?

```suggestion
            result.Should().ContainSingle();
            result.Should().ContainEquivalentTo(expected[0]);
```